src/configuration/iconfiguration.ts

```diff
@@ -398,4 +398,9 @@ export interface IConfiguration {
    * and `>>` with a linewise operator.
    */
   startofline: boolean;
+
+  /**
+   * Enable showing mark(s) in the gutter.
```

Comment:
This should match the description in `package.json` ("Show the currently set mark(s) in the gutter.")